docker/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM julia:1.7.2-bullseye
+
+WORKDIR /app
+COPY . /app
+COPY docker/PlutoDeployment.toml /etc/sliderserver/PlutoDeployment.toml
+
+# julia installation
```

Comment:
```suggestion
```
Comment is incorrect and not needed anyway. The project is "installed" not Julia.